cdi-core/src/main/java/com/linkedin/cdi/connection/S3Connection.java

```diff
@@ -84,43 +80,47 @@ public S3Connection(State state, JobKeys jobKeys, ExtractorKeys extractorKeys) {
   public WorkUnitStatus execute(WorkUnitStatus status) {
     s3Client = getS3HttpClient(getState());
 
-    String finalPrefix = getWorkUnitSpecificString(s3SourceV2Keys.getPrefix(), getExtractorKeys().getDynamicParameters());
+    JsonObject dynamicParameters = getExtractorKeys().getDynamicParameters();
```

Comment:
after running ./gradlew build the file is auto formatted. I extract some variable to improve readability.